easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'tensorflow-probability'
+version = '0.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/probability'
+description = """TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('TensorFlow', '2.1.0', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
```

Comment:
@dithwick We usually set `sanity_pip_check` "top-level" below (near `sanity_check_commands`):

```python
exts_default_options = {'source_urls': [PYPI_SOURCE]}

use_pip = True

exts_list = [
    ...
]

sanity_check_commands = ["python -c 'import tensorflow; import tensorflow_probability'"]

sanity_pip_check = True
```